assets/js/components/settings/SettingsRenderer.js

```diff
@@ -30,8 +30,8 @@ const { useSelect, useDispatch } = Data;
 
 export default function SettingsRenderer( { slug, isOpen, isEditing } ) {
 	const storeName = `modules/${ slug }`;
-	const isDoingSubmitChanges = useSelect( ( select ) => select( storeName )?.isDoingSubmitChanges?.() );
-	const haveSettingsChanged = useSelect( ( select ) => select( storeName )?.haveSettingsChanged?.() );
+	const isDoingSubmitChanges = useSelect( ( select ) => select( 'core/modules' )?.isDoingSubmitChanges?.( slug ) );
```

Comment:
This should use a constant for the `STORE_NAME` (aliased to `CORE_MODULES`).
Also, the conditional `?` chaining isn't necessary here since we can rely on the selector existing now.

Reply:
This is still unaddressed.

Reply:
[Addressed here](https://github.com/google/site-kit-wp/pull/2525/files#diff-c7a8015e4e1004ea9e43de2e0a06e5b64345dedb0dac21dc94ef5a449969ac7eR32-R34)